build/common.ps1

```diff
@@ -1,3 +1,13 @@
+# Check if running in a non-interactive shell and do not import/prompt if we are
+if(![bool]([Environment]::GetCommandLineArgs() -like '-noni*')) {
```

Comment:
`build.ps1` doesn't allow any `noni*` arguments, and I can't find a similar argument in any of our ps1 or `build/vsts_build.yaml` file. Does powershell automatically add it when it detects a non-interactive session?